app_server/test/controllers/UserController.js

```diff
@@ -83,8 +125,48 @@ lab.experiment('UserController log in tests', function () {
     Router.inject({method: 'POST', url: '/api/users/login',
                    payload: {appId: '123456789',
                              accessToken: 'xyz'}}, function (res) {
-      Code.expect(res.statusCode).to.equal(401);
+      Code.expect(res.statusCode).to.equal(400);
       done();
     });
   });
+
+  lab.test('Valid login', function (done) {
+    facebookServer.start(function (err) {
+      Code.expect(err).to.not.be.null();
+
+      Facebook.FACEBOOK_API_URL = 'http://localhost:8888';
+
+      Router.inject({method: 'POST', url: '/api/users/login',
+                    payload: {appId: '123456789',
+                              accessToken: 'xyz'}}, function (res) {
```

Comment:
You probably intended to add another space before `payload` (and subsequently `accessToken`) here?
(likewise in `test/policies/Authenticator.js`)